dashboards/google-app-engine/gae-app.json

```diff
@@ -1078,47 +1078,68 @@
         "width": 24,
         "height": 15,
         "widget": {
+          "title": "App Versions",
           "timeSeriesTable": {
-            "dataSets": [
-              {
-                "timeSeriesQuery": {
-                  "timeSeriesFilter": {
-                    "filter": "metric.type=\"appengine.googleapis.com/system/instance_count\" resource.type=\"gae_app\"",
-                    "aggregation": {
-                      "alignmentPeriod": "60s",
-                      "perSeriesAligner": "ALIGN_MEAN",
-                      "crossSeriesReducer": "REDUCE_SUM",
-                      "groupByFields": [
-                        "resource.label.\"module_id\"",
-                        "resource.label.\"project_id\""
-                      ]
-                    }
-                  },
-                  "unitOverride": "",
-                  "outputFullDuration": true
-                },
-                "tableTemplate": "",
-                "minAlignmentPeriod": "60s"
-              }
-            ],
-            "metricVisualization": "NUMBER",
             "columnSettings": [
               {
                 "column": "module_id",
+                "displayName": "Service",
                 "visible": true
               },
               {
                 "column": "project_id",
+                "displayName": "Project",
                 "visible": true
               },
               {
                 "column": "value",
                 "visible": false
+              },
+              {
+                "column": "version_id",
+                "displayName": "Version",
+                "visible": true
+              },
+              {
+                "column": "monitored_resource_container",
+                "visible": true
+              },
```

Comment:
Could we have a display name on this?

Also, what does this column add?

Reply:
can remove.